website/docs/getting-started/getting-started-guide.md

```diff
@@ -0,0 +1,185 @@
+---
+sidebar_position: 2
+---
```

Comment:
You may need to check the index number here

Reply:
I have made use of `index: 2` for the` gettingstarted.md` and `index: 1` for the `install.md` so that users first can visit the installation section and then move on towards the getting started guide